### PR TITLE
use breakpoints from settings in container plugin

### PR DESCRIPTION
**Pull request by davidetruffo**
Hi!
Here is my proposal. for issue #131 
Keep in mind that i'm new to python, so most likely the additions i propose could be rewritten in a more pythonic way. If so please feel free to either do it or advice me about it :)

The "only" issue i've bumped into is that if i create elements using some breakpoints, and then i REMOVE the br from the settings, then it breaks. (the element glossary is still having them stored, but i'm not sure how to proceed in this sense).

As a side note, i Have NOT touched the documentation, as I'm not sure if you'd like me to do it, or if you'd rather have control over it. Either way, let me know so i can gladly update it as well.

### CODE REVIEW

**Comment by jrief** [comment]
This PR is good.
I will merge this, as soon as I have some more time.

**Comment by rfleschenberg** [comment]
Closing and re-opening, to force a Travis build.

**Comment by davidetruffo** [comment]
I've added the dynamic breakpoints thing for the carousel plugin as well

**Comment by rfleschenberg** [comment]
This seems to have broken two tests: https://travis-ci.org/jrief/djangocms-cascade/jobs/113367819

**Comment by davidetruffo** [comment]
@rfleschenberg I'm not an expert, but as far as i can tell, the tests are failing on a ButtonWrapperPluginTest. 
I haven't touched anything related to any Button.
Also, the original PR failed as well, and then it was forced..

Could you please give me some hints about how to proceed? :)

**Comment by rfleschenberg** [comment]
@davidetruffo I think you are right, it looks like the test failures are actually unrelated to this PR. I will try to fix our Travis build to be more deterministic, and then I think we can merge this PR. Thanks!

File: cmsplugin_cascade/bootstrap3/container.py [hunk 1]
```diff
@@ -42,17 +46,25 @@ class BootstrapContainerPlugin(BootstrapPluginBase):
...
     form = BootstrapContainerForm
-    WIDGET_CHOICES = (
-        ('xs', _("Tiny (<{sm[0]}px)".format(**BS3_BREAKPOINTS))),
-        ('sm', _("Small (≥{sm[0]}px and <{md[0]}px)".format(**BS3_BREAKPOINTS))),
-        ('md', _("Medium (≥{md[0]}px and <{lg[0]}px)".format(**BS3_BREAKPOINTS))),
-        ('lg', _("Large (≥{lg[0]}px)".format(**BS3_BREAKPOINTS))),
-    )
+    breakpoints = list(BS3_BREAKPOINTS)
+    i = 0
+    widget_choises = []
```
Comment (rfleschenberg) [inline cmsplugin_cascade/bootstrap3/container.py]: Small typo here: I think this variable should be named `widget_choices`?

**Comment by davidetruffo** [comment]
Thank YOU! The pleasure of collaborating is mine :)

File: cmsplugin_cascade/bootstrap3/container.py (hunk 1 above)
Comment (davidetruffo) [inline cmsplugin_cascade/bootstrap3/container.py]: Most likey :)
Cold you fix it, or should i?

Comment (rfleschenberg) [inline cmsplugin_cascade/bootstrap3/container.py]: I think it's easiest if you update the PR. You can just push a new commit to your branch, and Github should automatically update the PR.

Comment (davidetruffo) [inline cmsplugin_cascade/bootstrap3/container.py]: I've just pushed it.

**Comment by jrief** [comment]
@davidetruffo please have a look at the way ``initial_height`` are implemented now. You should get familiar with dict comprehensions and enumerate, so reduce the code size.

**Comment by davidetruffo** [comment]
@jrief i told you i'm not an expert ;) 
I'll make a treasure of those hints! Thanks!

**Pull request merged**